_posts/2019-03-22-java-darmowa-czy-nie.md

```diff
@@ -0,0 +1,57 @@
+---
+layout:    post
+title:     "Java darmowa, czy nie?"
+date:      2019-03-28 08:00:00 +0100
```

Comment:
Zmieniłbym datę tutaj i w nazwie pliku, bo jeśli jutro zmerge'ujemy to się nie wygeneruje dobrze lista artykułów i w linku będzie zła data